src/Gui/DlgGeneral.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>425</width>
-    <height>578</height>
+    <height>767</height>
```

Comment:
I know this is an annoying behavior of Qt creator to ass space despite not needed. Qt Designer does this better.

However, be aware of this. You option does not need 200 px extra space and this can cause troubles for users with smaller screens (like me ;-) 

I will merge and fix this right now.

Reply:
@donovaly Thx for tip (and fix). Will keep an eye on this next time.